### PR TITLE
Dan Medvedev_HW1

### PR DESCRIPTION
**Pull request by danmedv**
Trying to update my old version. Not sure how to do it :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
-# hw01
\ No newline at end of file
+# hw01
+### Hi everyone,
+My name is Dan Medvedev and I'm from Moscow, Russia. ![](download.jpeg).
```
Comment (CommodoreDave) [inline README.md:3]: Gorgeous photo!

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
...
+
+I'm a second-year PhD student in *Behavioral Science*. One of the things I do research on is how people's moral evaluations differ based on their environments and cultural backgrounds. 
+Before starting my PhD at UChicago I lived in 
+* New York City, NY 
+* Ithaca, NY
```
Comment (CommodoreDave) [inline README.md:8]: I can only imagine that Ithaca is _thrilled_ to be on this list!

**Review by CommodoreDave — COMMENTED** [review]
A straightforward submission to a straightforward assignment! Could have included a link to really go for a gold star, but the guidelines for the homework suggested a minimum of 4 of the 5 elements listed, and that was certainly achieved here.

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** | X          |              |            |
| **Ease of access**          |         X  |              |            |

Remarks:

* It is great practice to commit regularly like you did! 
* You completed all the HW requirements. Great job :)
* For future reference, you don't need README.html, .Rproj, .DS_Store. You can look more into the class website of what doesn't need to be add to github.

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** | X          |              |            |
| **Ease of access**          |         X  |              |            |

Remarks:

* It is great practice to commit regularly like you did! 
* You completed all the HW requirements. Great job :)
* For future reference, you don't need README.html, .Rproj, .DS_Store. You can look more into the class website of what doesn't need to be add to github.

**Pull request closed without merging**